project/src/main/kotlin/org/ionproject/core/common/interceptors/LoggerInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package org.ionproject.core.common.interceptors
+
+import org.ionproject.core.common.Logger
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter
+import java.lang.Exception
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+
+class LoggerInterceptor : HandlerInterceptorAdapter() {
+
+    override fun preHandle(request: HttpServletRequest, response: HttpServletResponse, handler: Any): Boolean {
+        val startTime = System.currentTimeMillis()
+        Logger.logInfo(
+                "IP:${request.remoteAddr} | Method:${request.method} | Endpoint:${request.requestURI} | Timestamp:${startTime}"
+        )
+        request.setAttribute("startTime", startTime)
+        return true
+    }
+
+    override fun afterCompletion(request: HttpServletRequest, response: HttpServletResponse, handler: Any, ex: Exception?) {
+        val startTime : Long = request.getAttribute("startTime") as Long
+        val endTime : Long = System.currentTimeMillis()
+        Logger.logInfo(
+                """
+                    Total time taken to proccess request in milliseconds: ${endTime-startTime} ms
+                """.trimIndent()
+        )
+    }
```

Comment:
Why are we logging this? Just to help on the developement phase? I can see logging exceptions something important however.